apps/admin/backend/src/legacy_cast_vote_records.ts

```diff
@@ -113,6 +115,40 @@ export async function listCastVoteRecordFilesOnUsb(
 
   for (const entry of fileSearchResult.ok()) {
     if (entry.type === FileSystemEntryType.Directory) {
+      /* c8 ignore start */
+      if (
+        isFeatureFlagEnabled(
+          BooleanEnvironmentVariableName.ENABLE_CONTINUOUS_EXPORT
+        )
+      ) {
+        const directoryNameComponents =
+          parseCastVoteRecordReportExportDirectoryName(entry.name);
+        if (!directoryNameComponents) {
+          continue;
+        }
+        const metadataResult = await readCastVoteRecordExportMetadata(
+          entry.path
+        );
+        if (metadataResult.isErr()) {
+          continue;
+        }
+        const metadata = metadataResult.ok();
+        castVoteRecordFileMetadataList.push({
+          cvrCount: metadata.castVoteRecordReportMetadata.vxBatch
+            .map((batch) => batch.NumberSheets)
+            .reduce((sum, n) => sum + n, 0),
+          exportTimestamp: new Date(
+            metadata.castVoteRecordReportMetadata.GeneratedDate
+          ),
+          isTestModeResults: directoryNameComponents.inTestMode,
+          name: entry.name,
+          path: entry.path,
+          scannerIds: [directoryNameComponents.machineId],
+        });
+        continue;
+      }
```

Comment:
Whereas previously, we could extract everything that we needed for the "list of exports on the USB" UI from the directory name, now, we have to pull some info from the metadata file.

Note that I think it's okay for us to extract this information without authenticating the export (which can be time consuming since we have to re-hash), as we aren't actually importing anything yet. If someone tries to trick the system by changing a directory name or the metadata file, the worst they'll be able to do is get the "list of exports on the USB" UI to display something misleading. Importing will still fail.

We've applied similar principles to Java Card auth (e.g. [here](https://docs.google.com/document/d/17qLbrHYkjlEtXx18Oh1TovZz1D1dQd3Ze7azknqd3jA/edit#bookmark=id.gf0zmlln19uj)).